src/utils/files.py

```
@@ -0,0 +1,226 @@
+            ],
+        }
+
+        all_message_data.append(
+            {
+                "message_create": message_create,
+                "file_metadata": file_metadata,
+            }
+        )
+
+    if not all_message_data:
+        raise FileProcessingError()
```

Ruff (0.12.2)

Unnecessary parentheses on raised exception

Remove unnecessary parentheses

(RSE102)